### DISCUSSION TITLE
Scroll Position Goes Crazy When Window Resized

### DISCUSSION THREAD

**Discussion by blitzve0** [idiotWu/smooth-scrollbar#463] [Q&A]
### Checks

- [X] Not a duplicate.

### Version

8.7.4

### Description

**Summary**
I am using this on my website, When I Resize the Window, The Scroll Position behaves unexpectedly, I have shared video of my issue.
 
**Expected Behaviour**
Scroll Position should stay intact when resizing windows

https://user-images.githubusercontent.com/67477131/160181916-7a5f2a7a-6d01-4754-84aa-1ad6604e0de4.mp4

**Comment by dolphin-wood** [comment]
Duplicate of #416?
